en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -97,8 +97,9 @@ This can be difficult with receivers from different manufacturers due to the dif
 
 The following items should be checked during setup:
 
-* Verify that data for the second receiver is present. 
-  This will be logged as `vehicle_gps_position_1` and can also be checked when connected via *nsh console* using the `listener` command.
+* Verfiy that data for the second receiver is present. 
+This will be logged as vehicle_gps_position_1 and can also be checked when connected via the nsh console using the 'listener vehicle_gps_position -i 1' command. 
```

Comment:
You can use backticks around things like `vehicle_gps_position_1`  to mark them up as code. This usually makes things more readable. Here you should certainly mark up the command. I would do this like:

This will be logged as `vehicle_gps_position_1` and can also be checked when connected via the nsh console using the `listener vehicle_gps_position -i 1` command. 

OR 

This will be logged as `vehicle_gps_position_1` and can also be checked when connected via the nsh console using the command:
```bash
listener vehicle_gps_position -i 1
```